readme.md

```diff
@@ -27,8 +27,8 @@ The page should match the design Pixel Perfect: all the sizes, colors and distan
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://Mariiaantoniv.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://Mariiaantoniv.github.io/layout_moyo-header/report/html_report/)
```

Comment:
Add these links to the PR description
<img width="943" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/b11625f0-58d3-4c73-aee0-37caa93f95d1">